.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+Replace this paragraph with a description of what this PR is doing. Include
+links to relevant issues. If you're modifying existing behavior, describe the
+existing behavior, how this PR is changing it, and what motivated the change. If
+you're changing visual properties, consider including before/after screenshots
+(and runnable code snippets to reproduce them).
+
```

Comment:
I'd just add a separate header for related issues so it prompts people to find context before writing PRs